.dockerignore

```
@@ -0,0 +1,9 @@
+/.github
+/.pytest_cache
+/.venv
+/.vscode
+/.ansible
+.env*
+.gitignore
+Dockerfile
+README.md
```

Another suggestion here:
```suggestion
README.md
**/__pycache__
```

The explicit use of `/` at the start makes me think you might already be conscious of the tripping point but worth saying in case, that the syntax is slightly different to `.gitignore`; in particular that `.github` for Dockerignore means `/.github`, whereas for Gitignore that would be equivalent to `**/.github`